service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -85,13 +87,27 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             .setChild(child);
 
         caseProperties.put("placements", placementService.setPlacement(caseData, placement));
-        removeTemporaryFields(caseDetails, "placement", "placementChildName", "singleChild");
+        removeTemporaryFields(caseDetails, "placementChildName", "singleChild");
```

Comment:
Are you going to want to remove placement in the end or is it permanently staying?

Reply:
It's necessary in submitted event and we cannot modify case data there. It's not the prettiest solution but it's overwritten in mid-event. So yeah, it's staying like this.

Reply:
The send document event operates on a `about-to-submit` event though so it could be removed there couldn't it

Reply:
Wanted to do that at first, but decided it's not relevant place to do so. Send document event is not always caused by placement, in fact it's only 1 out of 7 orders triggering it. I would keep it in case data so it doesn't confuse anyone.

Reply:
Fair enough, clearly we need a new internal event just to remove this one property 😆